pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -127,6 +127,8 @@ var _ = BeforeEach(func() {
 	}
 	fakeClock.SetTime(time.Now())
 	deprovisioningController = deprovisioning.NewController(fakeClock, env.Client, provisioner, cloudProvider, recorder, cluster)
+	// Reset Feature Flags to test defaults
+	ctx = settings.ToContext(ctx, test.Settings(test.SettingsOptions{DriftEnabled: true}))
```

Comment:
This comment doesn't match with what the line is doing. Are you intending to just set the FeatureFlag for the drift cases. If so, then this should be set in a `BeforeEach` inside of the Drift `Describe` block